io.github.gssh_linux.gssh.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.gssh_linux.gssh
+runtime: org.gnome.Platform
+runtime-version: '44'
+runtime-options:
+sdk: org.gnome.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --env=GI_TYPELIB_PATH=/app/lib/girepository-1.0
```

Comment:
This should be set inside the sandbox:

```
>>> GIRepository.Repository.get_search_path()
['/app/lib/girepository-1.0', '/usr/lib/x86_64-linux-gnu/girepository-1.0']
```

Reply:
https://github.com/GNOME/gnome-build-meta/blob/1c4e2a340a62576f2ffc7b8f2ea9a589a9269036/elements/flatpak/platform.bst#L24

Reply:
Yes, it was not needed so I removed it.